highlight.io/components/QuickstartContent/mobile/react-native.tsx

```diff
@@ -0,0 +1,392 @@
+import { siteUrl } from '../../../utils/urls'
+import { QuickStartContent } from '../QuickstartContent'
+
+export const ReactNativeContent: QuickStartContent = {
+	title: 'React Native (beta)',
+	subtitle:
+		'Learn how to set up highlight.io errors, logs, and traces in your React Native application using OpenTelemetry.',
+	logoUrl: siteUrl('/images/quickstart/react.svg'),
+	entries: [
+		{
+			title: 'Install the OpenTelemetry npm packages.',
+			content:
+				'Install the following npm packages from `@opentelemetry` in your terminal.',
+			code: [
+				{
+					key: 'npm',
+					text: `# with npm
+npm install @opentelemetry/api @opentelemetry/core @opentelemetry/resources @opentelemetry/sdk-trace-base`,
+					language: 'bash',
+				},
+				{
+					key: 'yarn',
+					text: `# with yarn
+yarn add @opentelemetry/api @opentelemetry/core @opentelemetry/resources @opentelemetry/sdk-trace-base`,
+					language: 'bash',
+				},
+				{
+					key: 'pnpm',
+					text: `# with pnpm
+pnpm add @opentelemetry/api @opentelemetry/core @opentelemetry/resources @opentelemetry/sdk-trace-base`,
+					language: 'bash',
+				},
+			],
+		},
+		{
+			title: 'Create the OpenTelemetry tracer.',
+			content: `Some OpenTelemetry packages can't be used with the React Native's bundler, metro, due to some
+browser compatibility issues. As a work around, we created a custom exporter to serialize the data. A bundler-based
+solution is also in progress`,
+			code: [
```

Comment:
i'd describe here that the code should be pasted into separate files and name them, eg `highlight.ts`